README.md

```diff
@@ -67,6 +69,57 @@ There are many many tasks that collaborators do dfrom time to time and this
 project tries to make it as easy as possible or even automate stuff where it's
 suitable.
 
+## Usage
+
+You can currently access a beta version at [voxpupu.li](https://voxpupu.li).
+This is a MVP that we redeploy with enhancements every few days at the moment.
+The app uses GitHubs OAuth for authentication:
+
+[![login screenshot](login.png)](https://voxpupu.li)
+
+The application is developed by
+[Robert 'flipez' Müller](https://github.com/flipez) and
+[Tim 'bastelfreak' Meusel](https://github.com/bastelfreak). The current OAuth
+app is registed to their personal account but will soon be migrated to the Vox
+Pupuli GitHub organisation. You do not need to grant the application access to
+any repository. This OAuth setup is only used to authenticate the user.
+
+In the future it's possible to restrict the login or certain features to people
+that are members of specific GitHub organisations or teams. We didn't want to
+reimplement a whole usermanagement so we rely on GitHub OAuth.
+
+After the login, you see the following page:
+
+[![startpage screenshot](startpage.png)](https://voxpupu.li)
+
+Each yak shaving task is a row. It's prefixed with the amount of modules that
+are in this category. For example:
+
+[![single yak shaving action](shave.png)](https://voxpupu.li)
+
+You can click on each PR. The app displays all the informaiton that GitHub
+provides. It will also list open pull requests. It will be possible to filter
+this. The filtering will also work for all open pull requests in a namespace.
+The design and scope is currently discussed and implemented in
+[Issue #4](https://github.com/voxpupuli/vox-pupuli-tasks/issues/4).
+
+Besides being an OAuth application, this Ruby on Rails website is also a
+registered GitHub Application. This means that GitHub sends notifications
+for user interactions to the Rails app. The app gets information about every
+new pull request, new label, new code or comments in a pull request and much
+more. We currently store those notifications in a self hosted
+[Sentry](#sentry). The displayed data in the frontend comes from polling the
+GitHub API and from analysing the notifications. In the future we all add more
+automisation to the app. That will be based on the notifications. Usecases for
```

Comment:
"Use cases" is two words. "That will be based on the notifications" should probably not be a sentence by itself.

"In the future we all add more automisation to the app, which will be based on the notifications."??